service/src/main/java/org/folio/rest/workflow/service/OkapiDiscoveryService.java

```diff
@@ -94,7 +91,7 @@ public Map<String, List<Handler>> getHandlers(String tenant, String id) throws I
   }
 
   public JsonNode getModules(String tenant) {
-    String url = okapiUrl + basePath + PROXY_TENANT_BASE_PATH + tenant + MODULES_SUB_PATH;
+    String url = okapiUrl + PROXY_TENANT_BASE_PATH + tenant + MODULES_SUB_PATH;
```

Comment:
Why is base path removed here?
Why is `okapi.camunda.base-path` removed?

This seems like it will create a regresson.
The base path is being used.

Reply:
It is an unnecessary property. There is no regression here and can be tested with https://github.com/TAMULib/fw-cli/pull/96.